app-scientific/fricas/autobuild/defines

```diff
@@ -0,0 +1,40 @@
+PKGNAME=fricas
+PKGSEC=math
+
+# Note: x11-lib is used in the HyperDoc.
+PKGDEP__SBCL="gawk x11-lib sbcl"
+PKGDEP__ECL="gawk x11-lib ecl"
+PKGDEP__AMD64="$PKGDEP__SBCL"
+
+# FIXME: Continue out of heap while build
+#PKGDEP__PPC64EL="$PKGDEP__SBCL"
+#PKGDEP__ARM64="$PKGDEP__SBCL"
+PKGDEP="$PKGDEP__ECL"
+
+PKGSUG="texlive emacs"
+
+# Note: In order to build the graphic examples, xvfb
+# must be used, it is in this package.
+#
+# FIXME: Disable now, since fricas2d have bug related to
+# "double free or corruption (!prev)", and I can't fix it:P
+#
+#BUILDDEP="xorg-server"
+
+PKGDES="An advanced computer algebra system"
```

Comment:
Advanced is purely subjective and not helpful as a description. Maybe try this:

General purpose computer algebra system